closed/src/java.base/share/classes/com/sun/crypto/provider/NativeChaCha20Cipher.java

```diff
@@ -849,6 +853,16 @@ int doUpdate(byte[] in, int inOff, int inLen, byte[] out, int outOff)
          */
         int doFinal(byte[] in, int inOff, int inLen, byte[] out, int outOff)
                 throws ShortBufferException, AEADBadTagException, KeyException;
+        
+
```

Comment:
Please remove this extra blank line (one blank between declarations is sufficient).
Also ensure there is no trailing whitespace on any line (e.g line 856).